docs/asciidoc/static/include/pluginbody.asciidoc

```diff
@@ -0,0 +1,1021 @@
+:branch:          1.5
+:ls_version:		  1.5.0.beta1
+
+= How to write your own Logstash {plugintype} plugin
+
+This HOWTO will walk you through the process of developing a new {plugintype}
+for Logstash. The workflow has changed as of version 1.5 of Logstash, so you'll
+want to follow these new and improved steps to contribute a new plugin!  Since
+Logstash 1.5, all plugins are self-contained ruby gems.  In all previous
+versions, plugins were part of the core Logstash distribution.  This change
+makes it possible to develop plugins independent of the Logstash core, and
+release updates independently as well!
+
+NOTE: Logstash is written in Ruby.  If you are unfamiliar with the Ruby programming
+language, an excellent quick-start guide is at
+https://www.ruby-lang.org/en/documentation/quickstart/.
+
+== Get started
+
+{getstarted}
+
+=== Create a new GitHub repo
+As of version 1.5 of Logstash, the code for each plugin lives in its own GitHub
```

Comment:
We've already stated the version info--I don't think it's necessary to repeat it. Breaking the steps out into a list would make it easier to see at a glance what you need to do:

Each Logstash plugin lives in its own GitHub repository. To create a new repository for your plugin:

. Log in to GitHub.
. Click the *Repositories* tab. You'll see a list of other repositories you've forked or contributed to. 
. Click the green *New* button in the upper right.
. Specify the following settings for your new repo:
** Repository name--a unique name of the form logstash-{plugintype}-pluginname.
** Public or Private--your choise, but the repository must be Public if you want to submit it as an official plugin.
** Initialize this repository with a README--enables you to immediately clone the repository to your computer.
. Click *Create Repository*. 

In general, we should avoid using quotes as emphasis. Use backticks for code, filenames, etc., and *bold* for UI elements you're telling the user to interact with.